### DISCUSSION TITLE
Node Groups

### DISCUSSION THREAD

**Discussion by Fingie** [Comfy-Org/ComfyUI#1520] [Ideas]
Node groups and creating custom reusable nested workflows is why I love nodes so much. There's something so satisfying about creating my own personalized node group that solves my specific recurring problem. It's a core feature of every node-based program and is sorely lacking in comfy. 

it would pretty much be the already established Group functionality condensed into one node with custom in and out arguments. I can't imagine that would be toooo hard to do.

The ability to save and share these node groups would be amazing as well. Please please add node groups pretty pleaseeee

**Comment by coreyryanhanson** [comment]
That is the change I'd love to see the most as well. There is a pull request being developed that I suspect will make it's way into ComfyUI eventually:
https://github.com/comfyanonymous/ComfyUI/pull/931
